### ISSUE TITLE
Support Arrow Flight RPC

### ISSUE THREAD

**Issue by quinnj** [JuliaData/Arrow.jl#186]

**Comment by cwiese** [comment]
seconded :-)

**Comment by nrontsis** [comment]
+1

**Comment by quinnj** [comment]
It's a bit involved because we need proper HTTP2 support in Julia, then we can do a proper gRPC server implementation, and then with protobuf support for services, we'd be ready to do arrow flight on top 😅 . There were some efforts started [here](https://github.com/grlap/gRPC.jl) for gRPC using nghttp2 binary, but I'm not sure how far that effort got.

**Comment by okartal** [comment]
Thanks @quinnj, are you also aware of these efforts: https://discourse.julialang.org/t/ann-protobuf-jl-1-0-0/85885

> **`Service`s and `RPC`s are not yet implemented**. We will focus on these in near future as a part of our effort to build native gRPC libraries for Julia.

**Comment by quinnj** [comment]
Yes, @Drvi is my colleague and we worked on the ProtoBuf.jl rewrite together.

**Comment by kawild** [comment]
Hi,
is this feature still on the roadmap?

We use Julia extensively to query InfluxDB and its new Version (3.0) will only support Arrow Flight clients and I really don´t want to fall back to Python because of this missing feature.

Thank you!
